src/core/public/_variables.scss

```diff
@@ -4,3 +4,7 @@ $osdHeaderOffset: $euiHeaderHeightCompensation;
 $osdHeaderBreadcrumbBlueBackground: #b9d9eb;
 $osdHeaderBreadcrumbGrayBackground: #d9e1e2;
 $osdHeaderBreadcrumbCollapsedLink: #002a3a;
+$osdHeaderBreadcrumbPacificSkyDarkestBackground: #163f66;
```

Comment:
Was there a style guide approved color that we are using for this?

Reply:
most of these colors are not part of EUI/OUI. these are new colors and eventually will be part of OUI theming. we will be removing these colors once we start working on OUI and breadcrumb will be part of OUI